contents/blog/mykonos-hackathon.md

```diff
@@ -0,0 +1,180 @@
+
+In future, we’d want to add more data sources and build it using Llama to avoid the need to send any data to external companies.
+
+### MykoLogs
+
+<BuiltBy people={["Paul D'Ambra", "Tom Owers", "Ted Kaemming"]} />
+
+MykoLogs is a logging product that integrates with the existing PostHog SDKs, bringing backend logs straight into a shiny new product on PostHog. The best part? You can link backend logs to session recordings through the user’s session ID – letting you debug what was happening on the backend during your user's API requests. MykoLogs plays nicely with all other PostHog products, meaning logs and session replay are now BFFs. Debugging has never been this breezy!
+
+### Our own programming language
+
+<BuiltBy people={["Marius Andra", "Anirudh Pillai"]} />
+
+What’s cooler than having your own query language? Having your own general purpose programming language, of course. That’s literally what we built: PostHog’s first ever programming language, Hog. 
+
+Things moved fast the week after Mykonos: we split up the existing product analytics team and built a new team to productize Hog. We plan to use Hog to build our CDP and messaging products, and might even pivot the entire company around it. It was just another PostHog hackathon, no big deal. Stay tuned for the aftermath.
+
+### RealTimeHog 3000
+
+<BuiltBy people={["James Greenhill", "Zach Waterfield", "Michael Matloka", "Brett Hoerner"]} />
+
+Seeing people using your product live boosts dopamine levels. Probably. PostHog does that – but right now we keep you waiting a bit. Before a PostHog event is available for querying, it goes through our ingestion pipeline, where users are identified and the events themselves enriched. The process takes seconds, rarely a minute – a delay imperceptible in analytical queries, but a dopamine decrease in the live view.
+
+Introducing RealTimeHog 3000, a [livestream](https://github.com/PostHog/livestream) service powering our first truly real-time view of user activity. The process is simple: 
+
+1. Consume the raw events from the same Kafka topic as our ingestion service. 
+2. Stream them to you ASAP using server-sent events. 
+
+The lack of person data is a fair trade-off, because it doesn’t get more live than this. User activity appears within milliseconds of happening on the other side of the world.
+
+Speaking of data being processed live, perhaps at some point you’ve wondered if PostHog actually scales, or just talks the talk. To dispel any doubts, we developed one extra feature: an anonymized stream of all events being captured globally, with only the geolocation included. Hundreds of thousands of events per minute, and they look great on a 3D globe on PostHog’s website, where each event is an arc from the user’s location to that of the relevant data center. Global scale, visualized for your pleasure.
+
+![PostHog's 2024 hackathon in Mykonos](https://res.cloudinary.com/dmukukwp6/image/upload/hackathon3_0be5647083.png)
```

Comment:
Don't think this adds much, especially without the context of why we had a video call at an in-person event. Would be better to stick to pics / clips of demos.